stock_inventory_verification_request/models/stock_slot_verification_request.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SlotVerificationRequest(models.Model):
+    _name = 'stock.slot.verification.request'
+    _inherit = 'mail.thread'
+
+    @api.model
+    def create(self, vals):
+        vals['name'] = self.env['ir.sequence'].next_by_code(
+            'stock.slot.verification.request') or ''
+        return super(SlotVerificationRequest, self).create(vals)
+
+    @api.multi
+    def _count_involved_moves(self):
+        for rec in self:
+            rec.involved_move_count = len(self.involved_move_ids)
+
+    @api.multi
+    def _count_involved_inv_lines(self):
+        for rec in self:
+            rec.involved_inv_line_count = len(self.involved_inv_line_ids)
+
+    name = fields.Char(string='Name', readonly=True)
+    inventory_id = fields.Many2one(comodel_name='stock.inventory',
+                                   string='Inventory Adjustment',
+                                   readonly=True)
+    inventory_line_id = fields.Many2one(comodel_name='stock.inventory.line',
+                                        string='Inventory Line',
+                                        readonly=True)
+    location_id = fields.Many2one(comodel_name='stock.location',
+                                  string='Location',
+                                  required=True)
+    state = fields.Selection(selection=[
+        ('wait', 'Waiting Actions'),
+        ('open', 'In Progress'),
+        ('cancelled', 'Cancelled'),
+        ('done', 'Solved')
+    ], string='Status', default='wait')
+    responsible_id = fields.Many2one(comodel_name='res.users',
+                                     string='Assigned to')
+    product_id = fields.Many2one(comodel_name='product.product',
+                                 string='Product', required=True)
+    notes = fields.Text('Notes')
+    involved_move_ids = fields.Many2many(
+        comodel_name='stock.move',
+        relation='slot_verification_move_involved_rel',
+        column1='slot_verification_request_id',
+        column2='move_id',
+        string='Involved Stock Moves')
+    involved_move_count = fields.Integer(compute=_count_involved_moves)
```

Comment:
The computed field method name should be a string, not the method itself.
This allows overriding the method without the need to override the field too.

Reply:
This is it.
When using `compute=_method_name`, Python links the method itself (the python object), and you'll need to override the field to set a new method if you want to override the method.
When using `compute='_method_name'`, Python sets the method name, and Odoo calls that method on the model (including automatic call of the overrides), instead of directly the method of your file.